### PR TITLE
Support shorthand format for guardrails in config

### PR DESCRIPTION
**Pull request by roh26it**

### CODE REVIEW

File: src/handlers/handlerUtils.ts [hunk 1]
```diff
@@ -784,6 +810,16 @@ export async function tryTargetsRecursively(
     currentTarget.requestTimeout = inheritedConfig.requestTimeout;
   }
 
+  if (currentTarget.inputGuardrails) {
```
Comment (VisargD) [inline src/handlers/handlerUtils.ts:525]: Should we also handle scenario where user might have sent both: before_request_hooks and input_guardrails?

File: .gitignore [hunk 2]
```diff
@@ -141,3 +141,4 @@ build
 .idea
 plugins/**/.creds.json
 plugins/**/creds.json
+src/handlers/test.ts
```
Comment (narengogi) [inline .gitignore]: is this intentional?

**Comment by roh26it** [comment]
We should also support ids as part of this. Will need to add that to the Portkey middleware

**Review by VisargD — DISMISSED** [review]

File: .gitignore (hunk 2 above)
Comment (VisargD) [reply .gitignore]: Reverted this change

**Pull request merged**